pkg/gui/presentation/commits.go

```diff
@@ -440,14 +440,19 @@ func displayCommit(
 		mark = fmt.Sprintf("%s ", willBeRebased)
 	}
 
+	authorLength := common.UserConfig.Gui.CommitAuthorShortLength
+	if fullDescription {
+		authorLength = common.UserConfig.Gui.CommitAuthorLongLength
+	}
 	var authorFunc func(string) string
-	switch common.UserConfig.Gui.CommitAuthorFormat {
-	case "short":
+	if authorLength < 2 {
```

Comment:
I'd extract this logic into the authors.go file cos it's not specific to commits (I'm specifically thinking about a branch of mine that shows authors against branches)

Reply:
Yeah, makes sense. See c6b384f1c58d.